core/lib/generators/spree/install/templates/config/initializers/spree.rb

```diff
@@ -28,4 +28,8 @@
 
 # Spree::Api::Dependencies.storefront_cart_serializer = 'MyRailsApp::CartSerializer'
 
+Rails.application.config.after_initialize do
+  Spree::Config = Spree::AppConfiguration.new
+ends
```

Comment:
`ends` --> `end`

Reply:
Fixed! 😅 
Also i decided to comment these lines so they will be used only if needed. WDYT

Reply:
hmm, I assume that we need that for each extensions which will be override `AppConfiguration` class

Reply:
@damianlegawiec could you please take a 👁️ on this ? I'm working currently on `spree_mail_settings` gem and this fix will help me, because I'm also have to override `AppConfiguration` class